src/commands/dev/mod.rs

```diff
@@ -88,7 +107,7 @@ fn get_path_as_str(uri: &Uri) -> String {
 
 fn preview_request(
     req: Request<Body>,
-    client: Client<HttpsConnector<HttpConnector>>,
+    client: HyperClient<HttpsConnector<HttpConnector>>,
```

Comment:
where is this called (she asked herself)

Reply:
`client.request(req)` on line `145`